src/components/PrimaryTitle/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./primary-title.scss"
+
+interface PrimaryTitleProps {
+    title: string,
+    additionalClass?: string
+}
+
+export const PrimaryTitle:React.FunctionComponent<PrimaryTitleProps> = ({ title, additionalClass }) => {
+    const classes = ["primary-title"]
+
+    if (additionalClass) {
```

Comment:
You may consider alternative technique to manipulate `className` conditionally (e.g. based on props):
```jsx
..
const titleClassName = [
   'primary-title',
   ...(additionalClass ? [`primary-title_${additionalClass}`] : [])
].join(' ')

return (
   <span className={titleClassName}>{title}</span>
)
```